joplin/base/models.py

```diff
@@ -696,6 +741,18 @@ class InformationPageRelatedDepartments(ClusterableModel):
         PageChooserPanel("related_department"),
     ]
 
+class OfficialDocumentPageRelatedDepartments(ClusterableModel):
+    page = ParentalKey(OfficialDocumentPage, related_name='related_departments', default=None)
+    related_department = models.ForeignKey(
+        "base.departmentPage",
+        on_delete=models.PROTECT,
+    )
+
+    panels = [
+        # Use a SnippetChooserPanel because blog.BlogAuthor is registered as a snippet
```

Comment:
this looks like a bunk comment from a branch of mine that got merged without correcting